### DISCUSSION TITLE
Hi

### DISCUSSION THREAD

**Discussion by NhutMinhIT** [community/community#121018] [GitHub Education]
### Hi everyone,

Hi

**Comment by NhutMinhIT** [comment]
OKee

**Comment by soysantana** [comment]
Hi, how are you

**Comment by ebndev** [comment]
Hi @NhutMinhIT,

Thanks for being a part of the GitHub Community, we're glad you're here!

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" for our Community Discussions. Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding. 

Best of luck!

**Reply by ebndev** [reply]
Also you are more than welcome to introduce yourself and get to know other community members in our [Introduction Thread](https://github.com/orgs/community/discussions/23204)!
